cmd/kops/toolbox_instance_selector.go

```diff
@@ -501,10 +501,10 @@ func decorateWithMixedInstancesPolicy(instanceGroup *kops.InstanceGroup, usageCl
 	}
 
 	generatedWithLabelKey := "kops.k8s.io/instance-selector"
```

Comment:
@olemarkus what do you think, should we remove this completely?

Reply:
My intention for this label was to show infrastructure operators within the AWS console that the group was generated with instance selector.  I thought it would be more helpful as a real tag rather than a node label for that reason.

Reply:
i understand @bwagner5 . Curious to know why that info will be useful for operators?

Reply:
@bwagner5 would it cause any issues if this label was removed?

Reply:
I don't think so, I guess it's not too useful. I was thinking it would be nice to know which ones were generated with instance-selector in-case there were instance types that weren't actually fit for the compute nodes that were getting included. But I suppose it doesn't really matter since you can just hand modify the list.